src/bit-systems/audio-debug-system.ts

```diff
@@ -168,7 +168,7 @@ export function audioDebugSystem(world: HubsWorld) {
       }
       if (idx >= maxDebugEmitters) return;
 
-      const audio = APP.audios.get(emitterEid)!;
+      const audio = APP.world.eid2obj.get(emitterEid)! as AudioObject3D;
```

Comment:
We should _not_ change how we're storing these audios until after the migration. 

`APP.audios` and the rest of the audio state maps have key types of `AEntity | EntityID` because they are used by both code paths. There's no advantage to changing these now, and it will be significantly easier to do (with confidence) after we remove `AEntity`s. Please reverse these changes.